microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/MicroProfileLanguageServer.java

```diff
@@ -68,6 +71,8 @@
 
 	private final MicroProfileExtensionSettings extensionSettings;
 
+	private final ScheduledExecutorService delayer;
```

Comment:
I would probably declare this inside shutdown() as I we're only really using it there for now.